config/default.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
       revisions: 'revisions',
       save: 'save',
     },
+    plugins: {
```

Comment:
Why do we need to add this in?
In Content Types, [if this doesn't exist, an empty object is passed in](https://github.com/punchcard-cms/content-types/blob/master/lib/content-types/merge.js#L10-L22) to plugabilly. If `content-types` breaks if this isn't available, that should be fixed there. If it's broken in plugabilly, should be fixed there